src/test/ui/nll/ty-outlives/impl-trait-captures.stderr

```diff
@@ -1,16 +1,21 @@
 warning: not reporting region error due to nll
-  --> $DIR/impl-trait-captures.rs:21:5
+  --> $DIR/impl-trait-captures.rs:22:5
    |
 LL |     x
    |     ^
 
 error[E0621]: explicit lifetime required in the type of `x`
-  --> $DIR/impl-trait-captures.rs:21:5
+  --> $DIR/impl-trait-captures.rs:20:1
    |
-LL | fn foo<'a, T>(x: &T) -> impl Foo<'a> {
-   |               - consider changing the type of `x` to `&ReEarlyBound(0, 'a) T`
-LL |     x
-   |     ^ lifetime `ReEarlyBound(0, 'a)` required
+LL |   fn foo<'a, T>(x: &T) -> impl Foo<'a> {
+   |   ^             - consider changing the type of `x` to `&ReEarlyBound(0, 'a) T`
```

Comment:
Like a similar comment elsewhere: this sort of thing is the reason I'm not really thrilled with `ui/` tests that use `-Z verbose`

I recognize its a pre-existing issue with the test, but I think it would clean things up if you took the `-Z verbose` out the test file. (Unless you think the output is serving a useful purpose here? Provides hint to compiler devs about the internal region structure in play?)